README.md

```diff
@@ -1,2 +1,33 @@
-# swift
-WIP: Validate Swedish social security numbers
+# Personnummer swift
+
+Small library to validate and format swedish social security numbers called "Personnummer".
+
+## Installation 
+
+### Cocoapods
+	
+	pod 'Personnummer', '~> 1.0.0'
+	
+## Usage
+
+    // Validate
```

Comment:
Wrap this in syntax highlighting blocks https://help.github.com/articles/creating-and-highlighting-code-blocks/

I think it should be 
<pre>```swift
code
```</pre>